package.json

```diff
@@ -9,18 +9,40 @@
   "main": "index.js",
   "scripts": {
     "dev": "webpack-dev-server --env.NODE_ENV=dev",
+    "dev-test": "webpack-dev-server --env.NODE_ENV=test --env.TEST_SCENE=player",
     "prod": "webpack --mode=production",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "karma start --single-run"
   },
   "author": "James Cote (Coteh)",
   "license": "ISC",
   "devDependencies": {
+    "@types/chai": "^4.1.7",
+    "@types/mocha": "^5.2.6",
```

Comment:
Also reminding myself to merge changes from `master` when done the tests, as #35 introduces `prettier` and the tests should be formatted as well before they're merged in.